### ISSUE TITLE
Update Dependencies

### ISSUE THREAD

**Issue by vaibhavsingh97** [nightwatchjs/html-reporter#40]
## Description

Update these packages to new version:

| Package                          | Current Version | New Version |
|----------------------------------|-----------------|-------------|
| @emotion/react                   | ^11.10.6        | ^11.11.3    |
| @emotion/styled                  | ^11.10.6        | ^11.11.0    |
| @mui/icons-material              | ^5.11.11        | ^5.15.11    |
| @mui/material                    | ^5.11.15        | ^5.15.11    |
| @radix-ui/react-accordion        | 1.1.1           | 1.1.2       |
| @radix-ui/react-dropdown-menu    | ^2.0.5-rc.2     | ^2.0.6      |
| @radix-ui/react-tabs             | 1.0.3           | 1.0.4       |
| @radix-ui/react-toggle           | 1.0.2           | 1.0.3       |
| @radix-ui/react-tooltip          | 1.0.5           | 1.0.7       |
| @types/node                      | ^18.15.11       | ^20.11.20   |
| @types/react                     | ^18.0.33        | ^18.2.58    |
| @types/react-dom                 | ^18.0.11        | ^18.2.19    |
| @types/styled-components         | ^5.1.26         | ^5.1.34     |
| @typescript-eslint/eslint-plugin | ^5.57.1         | ^7.0.2      |
| @typescript-eslint/parser        | ^5.57.1         | ^7.0.2      |
| babel-plugin-styled-components   | ^2.0.7          | ^2.1.4      |
| eslint                           | ^8.37.0         | ^8.57.0     |
| eslint-config-prettier           | ^8.8.0          | ^9.1.0      |
| eslint-plugin-prettier           | ^4.2.1          | ^5.1.3      |
| eslint-plugin-react              | ^7.32.2         | ^7.33.2     |
| framer-motion                    | ^10.10.0        | ^11.0.6     |
| fuse.js                          | ^6.6.2          | ^7.0.0      |
| prettier                         | ^2.8.7          | ^3.2.5      |
| prism-react-renderer             | ^1.3.5          | ^2.3.1      |
| react-error-boundary             | ^4.0.3          | ^4.0.12     |
| react-medium-image-zoom          | ^5.1.3          | ^5.1.10     |
| react-virtuoso                   | ^4.1.1          | ^4.7.1      |
| styled-components                | ^5.3.9          | ^6.1.8      |
| ts-node                          | ^10.9.1         | ^10.9.2     |
| typescript                       | ^5.0.3          | ^5.3.3      |
| vite                             | ^4.2.1          | ^4.5.0      |

## TODO
After updating these packages, run locally using the following steps: https://github.com/nightwatchjs/html-reporter#-development

**Comment by varunsh17** [comment]
I would like to take up on this issue, can I be assigned @vaibhavsingh97 ?

**Comment by vaibhavsingh97** [comment]
The issue is all yours @varunsh17

**Comment by varunsh17** [comment]
Updated the dependencies, but facing issue here are the images of the errors @vaibhavsingh97 
![image](https://github.com/nightwatchjs/html-reporter/assets/81704366/6706e1ea-2c9a-4751-895d-5a0004c2981c)


```
  Error: The following dependencies are imported but could not be resolved:

  prism-react-renderer/themes/nightOwl (imported by /home/varun/Code/html-reporter/src/components/LogView/index.tsx)
  prism-react-renderer/themes/github (imported by /home/varun/Code/html-reporter/src/components/CodeBlock/index.tsx)

Are they installed?
    at file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:45779:23
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:45187:38
Failed to resolve import "prism-react-renderer/themes/nightOwl" from "src/components/LogView/index.tsx". Does the file exist?
[BABEL] Note: The code generator has deoptimised the styling of /home/varun/Code/html-reporter/src/icons/NoResult.tsx as it exceeds the max of 500KB.
Failed to resolve import "prism-react-renderer/themes/github" from "src/components/CodeBlock/index.tsx". Does the file exist?
6:02:22 PM [vite] Internal server error: Failed to resolve import "prism-react-renderer/themes/nightOwl" from "src/components/LogView/index.tsx". Does the file exist?
  Plugin: vite:import-analysis
  File: /home/varun/Code/html-reporter/src/components/LogView/index.tsx:2:18
  16 |  }
  17 |  import Highlight, { defaultProps } from "prism-react-renderer";
  18 |  import theme from "prism-react-renderer/themes/nightOwl";
     |                     ^
  19 |  import { Fragment } from "react";
  20 |  import {
      at formatError (file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:44062:46)
      at TransformContext.error (file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:44058:19)
      at normalizeUrl (file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:41844:33)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:41998:47
      at async Promise.all (index 4)
      at async TransformContext.transform (file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:41914:13)
      at async Object.transform (file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:44352:30)
      at async loadAndTransform (file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:55026:29)
      at async viteTransformMiddleware (file:///home/varun/Code/html-reporter/node_modules/vite/dist/node/chunks/dep-52909643.js:64430:32)```

**Comment by vaibhavsingh97** [comment]
@varunsh17 This is happening because [prism-react-renderer](https://github.com/FormidableLabs/prism-react-renderer) removed a few exports in the new version. The expectation with this updated dependency is that one needs to update the code according to the latest recommendations.

**Comment by varunsh17** [comment]
Yup I am trying to do the changes accordingly.

**Comment by wisekiran** [comment]
Hi, I updated the packages to the new version and created  a pull request: https://github.com/nightwatchjs/html-reporter/pull/42
Please review

**Comment by vaibhavsingh97** [comment]
fixed by #42
